### PR TITLE
Remove wizard hint

### PR DESCRIPTION
**Pull request by TheOneRing**
Fixes: #9810

### CODE REVIEW

File: src/gui/newwizard/pages/serverurlsetupwizardpage.cpp [hunk 1]
```diff
@@ -17,20 +17,21 @@ ServerUrlSetupWizardPage::ServerUrlSetupWizardPage(const QUrl &serverUrl)
...
     _ui->logoLabel->setText(QString());
     _ui->logoLabel->setPixmap(Theme::instance()->wizardHeaderLogo().pixmap(200, 200));
 
-    // we already use a placeholder, but it may be overwritten by the theme
     if (!Theme::instance()->wizardUrlHint().isEmpty()) {
-        _ui->urlLineEdit->setPlaceholderText(Theme::instance()->wizardUrlHint());
+        _ui->connectTotoolButton->setText(tr("Connect to %1").arg(Theme::instance()->wizardUrlHint()));
+        connect(_ui->connectTotoolButton, &QToolButton::clicked, this, [this] {
+            _ui->urlLineEdit->setText(Theme::instance()->wizardUrlHint());
+            // TODO: trigger next...
```
Comment (TheOneRing) [inline src/gui/newwizard/pages/serverurlsetupwizardpage.cpp:31]: As far as I can tell there is no way I can trigger this?

Comment (fmoc) [reply src/gui/newwizard/pages/serverurlsetupwizardpage.cpp:31]: The pages are not in charge of this. At the moment, the controlling objects (the corresponding state and/or the controller) poll from the page whether the inputs are valid for instance. If you need such signalling, I guess the cleanest approach is to add a signal to the page, and connect it in the state. So far, however, this kind of functionality has not been needed.

I think UX wise, it would be better to have the button should just copy the value into the line edit, and have the user click `Next` themselves.

**Review by fmoc — CHANGES_REQUESTED** [review]
Not really a fan of this button. IMO this just clutters the user interface. A label with a "link" which copies the value into the line edit is less intrusive and does the trick as well.

**Comment by TheOneRing** [comment]
> We shouldn't encourage people to connect to demo.owncloud.com.

I agree it it was how it was done previously.
I'd suggest to remove the login hint all together but we should check in the themes repo for customers using `wizardUrlHint`

**Comment by fmoc** [comment]
Yeah, let's get rid of this branding option. I don't see lots of use cases for it either.

File: src/gui/newwizard/pages/serverurlsetupwizardpage.cpp [hunk 2]
```diff
@@ -17,22 +17,13 @@ ServerUrlSetupWizardPage::ServerUrlSetupWizardPage(const QUrl &serverUrl)
 
     _ui->urlLineEdit->setText(serverUrl.toString());
 
-    // first, we declare this object as an event filter
-    // this then allows us to make the controller the event filter of the current page
-    _ui->urlLineEdit->installEventFilter(this);
```
Comment (fmoc) [inline src/gui/newwizard/pages/serverurlsetupwizardpage.cpp:22]: This change appears unrelated to the rest.

**Review by fmoc — CHANGES_REQUESTED** [review]

File: src/gui/newwizard/pages/serverurlsetupwizardpage.cpp (hunk 2 above)
Comment (TheOneRing) [reply src/gui/newwizard/pages/serverurlsetupwizardpage.cpp:22]: Yes but as the class is not implementing an event filer....?

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/12727/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/serverlog.log

Screenshots:
- [Adding_account_with_self_signed_certificate_for_the_first_time.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Adding_account_with_self_signed_certificate_for_the_first_time.png)
- [Adding_account_with_stay_safe_advance_configuration.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Adding_account_with_stay_safe_advance_configuration.png)
- [Adding_account_with_virtual_files_enabled_through_advance_configuration.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Adding_account_with_virtual_files_enabled_through_advance_configuration.png)
- [Adding_account_with_wrong_credentials.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Adding_account_with_wrong_credentials.png)
- [Adding_multiple_accounts.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Adding_multiple_accounts.png)
- [Adding_normal_Account.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Adding_normal_Account.png)
- [Both_original_and_copied_folders_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Both_original_and_copied_folders_can_be_synced.png)
- [Collaborator_should_not_see_to_whom_a_file_folder_is_shared.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Collaborator_should_not_see_to_whom_a_file_folder_is_shared.png)
- [Delete_a_file.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Delete_a_file.png)
- [Delete_a_folder.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Delete_a_folder.png)
- [Disable_VFS.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Disable_VFS.png)
- [Enable_VFS.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Enable_VFS.png)
- [File_with_long_name_(233_characters)_is_blacklisted.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/File_with_long_name_(233_characters)_is_blacklisted.png)
- [File_with_long_name_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/File_with_long_name_can_be_synced.png)
- [File_with_spaces_in_the_name_can_sync.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/File_with_spaces_in_the_name_can_sync.png)
- [Filenames_that_are_rejected_by_the_server_are_reported.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Filenames_that_are_rejected_by_the_server_are_reported.png)
- [Invalid_system_names_are_synced_in_linux.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Invalid_system_names_are_synced_in_linux.png)
- [Many_subfolders_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Many_subfolders_can_be_synced.png)
- [Modify_orignal_content_of_a_file_with_special_character.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Modify_orignal_content_of_a_file_with_special_character.png)
- [Progress_indicator_should_not_be_visible_after_unselecting_the_password_protection_checkbox_while_sharing_through_public_link.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Progress_indicator_should_not_be_visible_after_unselecting_the_password_protection_checkbox_while_sharing_through_public_link.png)
- [Share_files_folders_with_special_characters_in_their_name.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Share_files_folders_with_special_characters_in_their_name.png)
- [Syncing_a_file_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Syncing_a_file_to_the_server.png)
- [Syncing_a_folder_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Syncing_a_folder_to_the_server.png)
- [Syncing_all_files_and_folders_from_the_server.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Syncing_all_files_and_folders_from_the_server.png)
- [Syncing_file_of_1_GB_size.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Syncing_file_of_1_GB_size.png)
- [Syncing_folders_each_having_500_files.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Syncing_folders_each_having_500_files.png)
- [Tabs_in_toolbar_looks_correct.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Tabs_in_toolbar_looks_correct.png)
- [User_(non-author)_can_not_share_to_a_group_to_which_the_file_folder_is_already_shared.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/User_(non-author)_can_not_share_to_a_group_to_which_the_file_folder_is_already_shared.png)
- [VFS_is_disabled_by_default.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/VFS_is_disabled_by_default.png)
- [Verify_one_empty_folder_with_a_length_longer_than_the_allowed_limit_will_not_be_synced.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Verify_one_empty_folder_with_a_length_longer_than_the_allowed_limit_will_not_be_synced.png)
- [Verify_pre_existing_folders_in_local_(Desktop_client)_are_copied_over_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Verify_pre_existing_folders_in_local_(Desktop_client)_are_copied_over_to_the_server.png)
- [Verify_that_you_can_create_a_subfolder_with_long_name.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/Verify_that_you_can_create_a_subfolder_with_long_name.png)
- [change_collaborator_permissions_of_a_file_&_folder.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/change_collaborator_permissions_of_a_file_&_folder.png)
- [collaborators_are_listed_in_chronological_order.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/collaborators_are_listed_in_chronological_order.png)
- [logging_out.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/logging_out.png)
- [login_after_loggin_out.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/login_after_loggin_out.png)
- [remove_an_account_connection.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/remove_an_account_connection.png)
- [remove_the_only_account_connection.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/remove_the_only_account_connection.png)
- [reshare_a_file_folder.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/reshare_a_file_folder.png)
- [search_for_users_with_minimum_autocomplete_characters.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/search_for_users_with_minimum_autocomplete_characters.png)
- [share_a_file_with_many_users.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/share_a_file_with_many_users.png)
- [share_file_and_folder_to_a_group.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/share_file_and_folder_to_a_group.png)
- [sharee_creates_a_file_and_a_folder_inside_a_shared_folder.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharee_creates_a_file_and_a_folder_inside_a_shared_folder.png)
- [sharee_deletes_a_file_and_folder_shared_by_sharer.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharee_deletes_a_file_and_folder_shared_by_sharer.png)
- [sharee_edits_content_of_files_shared_by_sharer.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharee_edits_content_of_files_shared_by_sharer.png)
- [sharee_edits_shared_files_and_again_try_to_edit_after_write_permission_is_revoked.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharee_edits_shared_files_and_again_try_to_edit_after_write_permission_is_revoked.png)
- [sharee_renames_the_shared_file_and_folder.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharee_renames_the_shared_file_and_folder.png)
- [sharee_tries_to_create_a_file_and_a_folder_inside_a_shared_folder_without_write_permission.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharee_tries_to_create_a_file_and_a_folder_inside_a_shared_folder_without_write_permission.png)
- [sharee_tries_to_delete_shared_file_and_folder_without_permissions.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharee_tries_to_delete_shared_file_and_folder_without_permissions.png)
- [sharee_tries_to_edit_content_of_files_shared_without_write_permission.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharee_tries_to_edit_content_of_files_shared_without_write_permission.png)
- [sharing_a_folder_by_public_link_with_"Uploader"_role_and_check_if_file_can_be_downloaded.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharing_a_folder_by_public_link_with_"Uploader"_role_and_check_if_file_can_be_downloaded.png)
- [sharing_file_and_folder_with_user_who_has_some_other_shares.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharing_file_and_folder_with_user_who_has_some_other_shares.png)
- [sharing_file_folder_with_a_user_that_has_special_characters_as_username.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharing_file_folder_with_a_user_that_has_special_characters_as_username.png)
- [sharing_of_a_file_by_public_link_and_deleting_the_link.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharing_of_a_file_by_public_link_and_deleting_the_link.png)
- [sharing_of_a_file_by_public_link_with_password_and_changing_the_password.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/sharing_of_a_file_by_public_link_with_password_and_changing_the_password.png)
- [simple_sharing_of_a_file_by_public_link_with_default_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/simple_sharing_of_a_file_by_public_link_with_default_expiration_date.png)
- [simple_sharing_of_a_file_by_public_link_with_password_and_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/simple_sharing_of_a_file_by_public_link_with_password_and_expiration_date.png)
- [simple_sharing_of_file_and_folder_by_public_link_with_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_with_expiration_date.png)
- [simple_sharing_of_file_and_folder_by_public_link_with_password.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_with_password.png)
- [simple_sharing_of_file_and_folder_by_public_link_without_password.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_without_password.png)
- [simple_sharing_of_folder_by_public_link_with_different_roles.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/simple_sharing_of_folder_by_public_link_with_different_roles.png)
- [simple_sharing_with_user.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/simple_sharing_with_user.png)
- [try_to_reshare_a_file_folder_shared_without_share_permission.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/try_to_reshare_a_file_folder_shared_without_share_permission.png)
- [try_to_self_share_a_file_folder.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/try_to_self_share_a_file_folder.png)
- [try_to_share_a_file_folder_with_a_user_to_whom_the_file_has_already_been_shared.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/try_to_share_a_file_folder_with_a_user_to_whom_the_file_has_already_been_shared.png)
- [unshare_a_reshared_file.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/unshare_a_reshared_file.png)
- [unshare_a_shared_file_and_folder.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/unshare_a_shared_file_and_folder.png)
- [various_types_of_files_can_be_synced_from_client_to_server.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/various_types_of_files_can_be_synced_from_client_to_server.png)
- [various_types_of_files_can_be_synced_from_server_to_client.png](https://cache.owncloud.com/public/owncloud/client/12727/guiReportUpload/screenshots/various_types_of_files_can_be_synced_from_server_to_client.png)

</details>

File: src/gui/newwizard/pages/serverurlsetupwizardpage.cpp (hunk 2 above)
Comment (fmoc) [reply src/gui/newwizard/pages/serverurlsetupwizardpage.cpp:22]: I'm not a pro at this, but if I remember this correctly, I had to install the window as an event filter manually, otherwise I would not have received the events upstream. If this is no longer necessary (or maybe never was), great!

**Review by fmoc — APPROVED** [review]
